contracts/scripts/utils.js

```diff
@@ -31,110 +31,93 @@ async function main() {
-  const tx = await contract.createCohort(cohortId, limit, merkleRoot, {
-    gasPrice: ethers.utils.parseUnits('100', 'gwei'),
-  });
-  console.log(tx);
-  await tx.wait();
-}
+};
 
-async function fetchCohort(cohortId) {
+/**
+ * @param string cohortId
+ * Fetches the cohort on the contract and prints out the Limit number of the cohort
+ */
+const fetchCohort = async (cohortId) => {
   const tx = await contract.cohorts(cohortId);
   console.log(tx);
   console.log('LIMIT:', tx[0].toNumber());
-}
+};
 
-async function fetchUri(tokenId) {
+/**
+ * @param number tokenId
+ * Fetches the URI of the given tokenId
+ */
+const fetchUri = async (tokenId) => {
   const tx = await contract.tokenURI(tokenId);
   console.log(tx);
-}
-
-async function updateMerkleRoot(cohortId, merkleRoot) {
+};
+
+/**
+ * @param string cohortId
+ * @param string merkleRoot
+ * @param string gasPrice
+ * Updates the merkle root with new root value for a given cohort
+ */
+const updateMerkleRoot = async (cohortId, merkleRoot, gasPrice) => {
   const tx = await contract.setMerkleRoot(cohortId, merkleRoot, {
-    gasPrice: ethers.utils.parseUnits('50', 'gwei'),
+    gasPrice: ethers.utils.parseUnits(gasPrice, 'gwei'),
   });
   console.log(tx);
   await tx.wait();
-}
-
-async function createCohort(cohort_id, limit) {
+};
+
+/**
+ * @param string cohort_id
+ * @param number limit
+ * @param string gasPrice
+ * Creates a new cohort on the contract with an initial merkle root
+ */
+const createCohort = async (cohort_id, limit, gasPrice) => {
   const tx = await contract.createCohort(
     cohort_id,
     limit,
     '0x0000000000000000000000000000000000000000000000000000000000000000',
     {
-      gasPrice: ethers.utils.parseUnits('100', 'gwei'),
+      gasPrice: ethers.utils.parseUnits(gasPrice, 'gwei'),
       gasLimit: 100000,
     }
   );
   console.log(tx);
   await tx.wait();
-}
-
-async function fetchOwner(address) {
-  const tx = await contract.claimed(
-    address,
-    'CH4f447780-07cf-408a-8f4c-253a8b4e8bae'
-  );
-  console.log(tx);
-  return tx;
-}
-
-//-- EXECUTION LAND --//
-
-// fetchOwner('0xE43973F95F344A1178CA661440E6A1F0d0a854B6')
-//   .then(() => process.exit(0))
-//   .catch((error) => {
-//     console.error(error);
-//   });
-
-// fetchUri(14629)
-//   .then(() => process.exit(0))
-//   .catch((error) => {
-//     console.error(error);
-//   });
-
-// const runAction = async () => {
-//   for (let i = 83854; i++; i < 83884) {
-//     await cancelTransaction(i);
-//     console.log('done');
-//   }
-// };
-
-// runAction();
-
-createCohort('CHa07ac83b-bc08-46b6-b561-abe05d5bda00', 1000)
+};
+
+/**
+ * @param string address
+ * @param string cohort_id
+ * Fetches the token number of a given cohort_id and address
+ */
+const fetchOwner = async (address, cohort_id) => {
+  const tx = await contract.claimed(address, cohort_id);
+  console.log(tx.toNumber());
+};
+
+// Execution Land
+createCohort('CHaee9050e-fdb6-4094-9097-7f4fa00b8fd6', 1000, '100')
```

Comment:
hmm.. hard coded id?

Reply:
This is just an example! Tweaked though :)